FHEM/59_Buienradar.pm

```diff
@@ -914,11 +1033,21 @@ sub ResetResult {
 }
 
 sub Debugging {
```

Comment:
Die Idee finde ich gut, so einer Debugging und Error sub.

Das könnte man auch global in einem eigenen Modul gut gebrauchen :)

Reply:
`Debugging` ist eigentlich nur ein Wrapper, der [`Debug`](https://wiki.fhem.de/wiki/DevelopmentModuleAPI#Debug) aus `fhem.pl` ein bisschen komfortabler macht. An sich fehlt sowas wie strukturiertes Error-Handling ja weitesgehend im Projekt, leider.

Reply:
Ja , leider kann man den ja nicht in ein eigenes Perl Modul packen , da Log3 und Debug im Perl main liegen :(

Wenn ich das jetzt auch verwenden möchte, muss ich den Code kopieren.

Nicht falsch verstehen. Ich finde den wrapper gut und verstehe auch wieso Du ihn in deinem Modul untergebracht hast.

Reply:
Ne, hab's nicht falsch verstanden 👍 
Im Prinzip wäre das was für [Utils.pm](https://svn.fhem.de/trac/browser/trunk/fhem/FHEM/99_Utils.pm) oder?